src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/classes/string_tensor_pack.cpp

```diff
@@ -0,0 +1,130 @@
+        inputs.insert({ funcInputs[0].get_node_shared_ptr(), data_tensor });
+    }
+
+void StringTensorPackLayerCPUTest::SetUp() {
+        StringTensorPackLayerTestParams basicParamsSet;
+        CPUSpecificParams cpuParams;
+        std::tie(basicParamsSet, cpuParams) = this->GetParam();
+        std::tie(inFmts, outFmts, priority, selectedType) = cpuParams;
+
+        StringTensorPackSpecificParams StringTensorPackParams;
+        ElementType indicesPrecision;
+        std::tie(StringTensorPackParams, indicesPrecision, targetDevice) = basicParamsSet;
+
+        InputShape indicesShape;
+        InputShape symbolsShape;
+        std::vector<int64_t> begins;
+        std::vector<int64_t> ends;
+        std::tie(indicesShape, symbolsShape, begins, ends) = StringTensorPackParams;
+
+        init_input_shapes({indicesShape, indicesShape, symbolsShape});
+        auto symbolsParameter = std::make_shared<ov::op::v0::Parameter>(ov::element::u8, inputDynamicShapes[2]);
+        auto beginsConst = std::make_shared<ov::op::v0::Constant>(indicesPrecision, indicesShape.second.front(), begins);
+        auto endsConst = std::make_shared<ov::op::v0::Constant>(indicesPrecision, indicesShape.second.front(), ends);
+        auto StringTensorPack = std::make_shared<ov::op::v15::StringTensorPack>(beginsConst, endsConst, symbolsParameter);
+
+        ov::ParameterVector params{ symbolsParameter };
+        function = makeNgraphFunction(ov::element::string, params, StringTensorPack, "StringTensorPack");
+}
+
+TEST_P(StringTensorPackLayerCPUTest, CompareWithRefs) {
+    run();
+    CheckPluginRelatedResults(compiledModel, "StringTensorPack");
+}
+
+const std::vector<StringTensorPackSpecificParams> StringTensorPackParamsVector = {
+    StringTensorPackSpecificParams {
+        InputShape{{}, {{3}}},
+        InputShape{{}, {{9}}},
+        std::vector<int64_t>{1, 4, 6},
+        std::vector<int64_t>{4, 6, 8}
+    },
+    StringTensorPackSpecificParams {
+        InputShape{{}, {{1, 3, 4}}},
+        InputShape{{}, {{108}}},
+        std::vector<int64_t>{7, 28, 30, 30, 41, 42, 50, 50, 74, 76, 80, 92},
+        std::vector<int64_t>{28, 30, 30, 41, 42, 50, 50, 74, 76, 80, 92, 108}
+    },
+    StringTensorPackSpecificParams {
+        InputShape{{}, {{1, 1, 1, 2}}},
+        InputShape{{}, {{67}}},
+        std::vector<int64_t>{30, 31},
+        std::vector<int64_t>{31, 31}
+    },
+    StringTensorPackSpecificParams {
+        InputShape{{-1, -1, -1}, {{1, 3, 4}}},
+        InputShape{{-1}, {{108}}},
+        std::vector<int64_t>{7, 28, 30, 30, 41, 42, 50, 50, 74, 76, 80, 92},
+        std::vector<int64_t>{28, 30, 30, 41, 42, 50, 50, 74, 76, 80, 92, 108}
+    },
+    StringTensorPackSpecificParams {
+        InputShape{{-1, {1, 4}, {2, 3}}, {{1, 2, 2}}},
+        InputShape{{{50, 100}}, {{67}}},
+        std::vector<int64_t>{30, 31, 56, 60},
+        std::vector<int64_t>{31, 56, 60, 67}
```

Comment:
Please add at least 2 different target shapes to test dynamic shapes change.

Reply:
Done.